sys/posix/pthread/pthread.c

```diff
@@ -38,10 +38,10 @@
 #define ENABLE_DEBUG (0)
 
 #if ENABLE_DEBUG
-#   define PTHREAD_REAPER_STACKSIZE THREAD_STACKSIZE_MAIN
+#   define PTHREAD_REAPER_STACKSIZE (THREAD_STACKSIZE_IDLE + THREAD_EXTRA_STACKSIZE_PRINTF)
```

Comment:
Have you tested, if this is enough?

Reply:
We found an issue on msba2. Fixing it now.

Reply:
It should be enough now.